### PR TITLE
SRC-6418  Noetic migration 2 of 2

### PR DESCRIPTION
**Pull request by maxzieba**
## Proposed changes

User Story: https://shadowrobot.atlassian.net/browse/SRC-6418
Migrating to noetic

## Checklist

If the task is applicable to this pull request (see applicability criteria in brackets), make sure it is completed before checking the corresponding box. Otherwise, tick the box right away. Make sure that **ALL** boxes are checked **BEFORE** the PR is merged.

- [ ] Manually tested that added code works as intended (if any functional/runnable code is added).
- [ ] Added automated tests (if a new class is added (Python or C++), interface of that class must be unit tested).
- [ ] Tested on real hardware (if the changed or added code is used with the real hardware).
- [ ] Added documentation (For any new feature, explain what it does and how to use it. Write the documentation in a relevant space, e.g. Github, Confluence, etc.)

### CODE REVIEW

File: sr_speech_control/src/sr_speech_control/speech_control.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
```
Comment (biofotis) [inline sr_speech_control/src/sr_speech_control/speech_control.py]: should we specify python3?

File: sr_utilities_common/src/sr_utilities_common/aws_manager.py [hunk 2]
```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python
```
Comment (biofotis) [inline sr_utilities_common/src/sr_utilities_common/aws_manager.py]: here as well

File: sr_utilities_common/test/aws_manager_test.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/python
```
Comment (biofotis) [inline sr_utilities_common/test/aws_manager_test.py]: and here

File: sr_wear_logger/CMakeLists.txt [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+#############
+
+# all install targets should use catkin DESTINATION variables
+# See http://ros.org/doc/api/catkin/html/adv_user_guide/variables.html
+
+## Mark executable scripts (Python etc.) for installation
+## in contrast to setup.py, you can choose the destination
+ catkin_install_python(PROGRAMS
+   src/sr_wear_logger/sr_wear_logger.py
+   DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
```
Comment (biofotis) [inline sr_wear_logger/CMakeLists.txt:162]: how about the launch file?

File: sr_wear_logger/src/sr_wear_logger/dummy_publisher.py [hunk 5]
```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
```
Comment (biofotis) [inline sr_wear_logger/src/sr_wear_logger/dummy_publisher.py]: python3?

File: sr_wear_logger/src/sr_wear_logger/sr_wear_logger.py [hunk 6]
```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/env python
```
Comment (biofotis) [inline sr_wear_logger/src/sr_wear_logger/sr_wear_logger.py]: as well

File: sr_wear_logger/test/test_sr_wear_logger.py [hunk 7]
```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/python
```
Comment (biofotis) [inline sr_wear_logger/test/test_sr_wear_logger.py]: here too

**Review by biofotis — CHANGES_REQUESTED** [review]

File: sr_wear_logger/src/sr_wear_logger/sr_wear_logger.py (hunk 6 above)
Comment (maxzieba) [reply sr_wear_logger/src/sr_wear_logger/sr_wear_logger.py]: Corrected

File: sr_wear_logger/test/test_sr_wear_logger.py (hunk 7 above)
Comment (maxzieba) [reply sr_wear_logger/test/test_sr_wear_logger.py]: Corrected

File: sr_wear_logger/src/sr_wear_logger/dummy_publisher.py (hunk 5 above)
Comment (maxzieba) [reply sr_wear_logger/src/sr_wear_logger/dummy_publisher.py]: Corrected

File: sr_utilities_common/test/aws_manager_test.py (hunk 3 above)
Comment (maxzieba) [reply sr_utilities_common/test/aws_manager_test.py]: Corrected

File: sr_utilities_common/src/sr_utilities_common/aws_manager.py (hunk 2 above)
Comment (maxzieba) [reply sr_utilities_common/src/sr_utilities_common/aws_manager.py]: Corrected

File: sr_speech_control/src/sr_speech_control/speech_control.py (hunk 1 above)
Comment (maxzieba) [reply sr_speech_control/src/sr_speech_control/speech_control.py]: Corrected

File: sr_wear_logger/CMakeLists.txt (hunk 4 above)
Comment (maxzieba) [reply sr_wear_logger/CMakeLists.txt:162]: Added in lines 187-189

**Review by biofotis — APPROVED** [review]

**Review by giusebar — APPROVED** [review]

**Pull request merged**